rainforest-cli.go

```diff
@@ -8,25 +8,33 @@ import (
 
 var apiToken string
 
+var smartFolderID int
+var siteID int
+var tags string
+
 var baseURL = "https://app.rainforestqa.com/api/1"
 var out io.Writer = os.Stdout
 
-func parseCommands() []string {
+func parseCommands(arguments []string) []string {
```

Comment:
Personally I think it would be a lot cleaner to parse all arguments in a helper function, e.g. `parseArgs`, that doesn't actually mutate `os.Args`. It would also make testing easier. So separate the args into two slices (commands and flags) and parse the flags using `flag.CommandLine.Parse(flags)` instead of `flag.Parse()`.